app/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -827,7 +834,9 @@ public void onZoomUpdate( float scale, Origination origin ) {
 
   @Override
   public void onZoomEnd( float scale, Origination origin ) {
-    mDetailLevelManager.unlockDetailLevel();
+    if ( origin == null ) {
```

Comment:
on end, do we care about origin?  should we always assume we should resume render...?